Jenkinsfile

```diff
@@ -25,31 +22,50 @@ pipeline {
         GIT_COMMITTER_EMAIL = "jenkins@jenkins.io"
     }
     
-    // The order that sections are specified doesn't matter - this will still be run
-    // after the stages, even though it's specified before the stages.
-    post {
-        // No matter what the build status is, run this step. There are other conditions
-        // available as well, such as "success", "failed", "unstable", and "changed".
-        always {
-            junit '*/target/surefire-reports/*.xml'
-        }
-        success {
-            archive "**/target/*.hpi"
-            archive "**/target/site/jacoco/jacoco.xml"
-        }
-        unstable {
-            archive "**/target/*.hpi"
-            archive "**/target/site/jacoco/jacoco.xml"
-        }
-    }
 
     stages {
         // While there is only one stage here, you can specify as many stages as you like!
         stage("build") {
+            agent {
+                label "java"
+            }
             steps {
                 sh 'mvn clean install -Dmaven.test.failure.ignore=true'
             }
+            post {
+                // No matter what the build status is, run this step. There are other conditions
+                // available as well, such as "success", "failed", "unstable", and "changed".
+                always {
+                    junit '*/target/surefire-reports/*.xml'
+                }
+                success {
+                    archive "**/target/*.hpi"
+                    archive "**/target/site/jacoco/jacoco.xml"
```

Comment:
why not use the jaccoco plugin?  Also generally better to run with and without coverage in a different lifecycle...
But this is not anything new so - Meh...

Reply:
Simple answer - at the time we added that, the jacoco plugin didn't have a `@Symbol` yet. =) But it does now, so I'll add that.

Reply:
Check that, it still hasn't been released yet.